src/multi_vector_simulator/D2_model_constraints.py

```diff
@@ -565,6 +583,60 @@ def prepare_constraint_minimal_degree_of_autonomy(
             )
             demand_list.append(asset)
 
+    # Preprocessing for log messages
+    demand_string = ", ".join(map(str, demand_list))
+
+    logging.debug(f"Demands: {demand_string}")
```

Comment:
Yes it is. 
The way the preparation function worked before the splitting the log messages were clearer / better to find in the code. However, I found it better to split the function instead of splitting the code. 
Another option here could be to use a parameter that is used in the preparation function's log message, e.g. `constraint_str="minimal degree of autonomy"`. If you don't react to this I won't change anything.

Reply:
It is working, so lets leave it.